experimental/tf_proxy/server/src/main.rs

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pub mod proto {
+    tonic::include_proto!("oak.session.stream.v1");
+}
+
+use crate::attestation::AttestationServer;
+use anyhow::Context;
+use proto::streaming_session_server::StreamingSessionServer;
+use std::{
+    fs::canonicalize,
+    net::{Ipv6Addr, SocketAddr},
+    path::PathBuf,
+};
+use structopt::StructOpt;
+use tonic::transport::Server;
+
+pub mod attestation;
+pub mod grpc;
+
+// The name of the UNIX domain socket for communicating with the TF model server.
+static SOCKET: &str = "/tmp/serving_uds";
+
+#[derive(StructOpt, Clone, Debug)]
+pub struct Opt {
+    #[structopt(
+        long,
+        default_value = "8080",
+        help = "Port number that the server listens on."
```

Comment:
Is it the backend server or ther proxy server?

Reply:
The proxy server. The backend server listens on UDS. Clarified the help string.